src/AppInstallerCLICore/ConfigurationDynamicRuntimeFactory.cpp

```diff
@@ -0,0 +1,202 @@
+
+                return m_currentIntegrityLevel;
+            }
+
+            // Serializes the set properties to be sent to the remote server
+            std::string SerializeSetProperties()
+            {
+                // TODO: Serialize any properties to JSON that do not get serialized to the file stream
+                //       Currently the only one is `Path`.
+                //       This should be the path to sending properties set via SetProcessorFactory::IPwshConfigurationSetProcessorFactoryProperties
+                return R"({
+  "filePath": "E:\Temp\PSGallery_NoModule_NoSettings.yml"
+})";
+            }
+
+            // Serializes a version of the set that only contains the units that require high integrity level
+            std::string SerializeHighIntegrityLevelSet()
+            {
+                // TODO: Extract only the units that require high integrity and place them in a new set, then serialize it
+                // TODO: Implement ConfigurationSet::Serialize
+                return R"(properties:
+    configurationVersion: 0.1
+    resources:
+      - resource: XmlFileContentResource
+        directives:
+          description: Set XML file contents
+          SecurityContext: elevated
+)";
+            }
+
+            ProcessorMap::iterator CreateSetProcessorForIntegrityLevel(Security::IntegrityLevel integrityLevel)
+            {
+                IConfigurationSetProcessorFactory factory;
+
+                // If we got here, the only option is that the current integrity level is not High.
+                if (integrityLevel == Security::IntegrityLevel::High)
+                {
+                    factory = CreateOutOfProcessFactory(true, SerializeSetProperties(), SerializeHighIntegrityLevelSet());
+                }
+                else
+                {
+                    THROW_WIN32(ERROR_NOT_SUPPORTED);
+                }
+
+                return m_setProcessors.emplace(integrityLevel, DynamicProcessorInfo{ factory, factory.CreateSetProcessor(m_configurationSet) }).first;
+            }
+
+            Security::IntegrityLevel m_currentIntegrityLevel;
+            ProcessorMap m_setProcessors;
+            ConfigurationSet m_configurationSet;
+        };
+
+        // This is implemented completely in the packaged context for now, if we want to make it more configurable, we will probably want to move it to configuration and
+        // have this implementation leverage that one with an event handler for the packaged specifics.
+        // TODO: Add SetProcessorFactory::IPwshConfigurationSetProcessorFactoryProperties and pass values along to sets on creation
+        //       In turn, any properties must only be set via the command line (or eventual UI requests to the user).
+        struct DynamicFactory : winrt::implements<DynamicFactory, IConfigurationSetProcessorFactory, winrt::cloaked<WinRT::ILifetimeWatcher>>, WinRT::LifetimeWatcherBase

```

Comment:
The object flow is:
`DynamicFactory` holds the same integrity level remote factory
`DynamicFactory` creates `DynamicSetProcessor`, giving it a copy of the same integrity level remote factory
`DynamicSetProcessor` creates more remote factories and remote set processors at other integrity levels as needed
`DynamicSetProcessor` uses the appropriate remote set processor to create the unit processor